packages/app/src/app/utils/executor-manager.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  IExecutor,
+  ServerExecutor,
+  SandboxExecutor,
+} from '@codesandbox/executors';
+import { IFiles } from '@codesandbox/executors/dist/executor';
+import { Sandbox } from '@codesandbox/common/lib/types';
+import getDefinition from '@codesandbox/common/lib/templates';
+import { getModulePath } from '@codesandbox/common/lib/sandbox/modules';
+import { generateFileFromSandbox } from '@codesandbox/common/lib/templates/configuration/package-json';
+
+function getExecutorType(isServer: boolean) {
```

Comment:
@CompuIves This can be an arrow function

```suggestion
const getExecutorType = (isServer: boolean) => {
```